Gruntfile.js

```diff
@@ -40,6 +40,27 @@ module.exports = function (grunt) {
         }
       }
     },
+    run: {
+      options: {
+      },
+      submodulesUpdate: {
+        cmd: 'git',
+        args: [
+          'submodule',
+          'update',
+          '--init'
```

Comment:
The reason I'd like to have the latest commit is so we don't have to wait for the next release of patternfly before we integrate some PRs. For example, Gabriel created a PR to add a Patternfly example to pf.org. Unfortunately, we cannot merge his PR this until the next release of patternfly, when the example's template becomes available.

Of course we can change the pf.org dependency to use the latest commit of Patternfly. However, that generates conflicts with the Travis build because angular-patternfly also depends on Patternfly. The work around for that is to add a resolve statement in bower.json, but that gets ugly quickly with all the dependency changes, lately. And the resolve statement must be removed when we bump the version numbers upon the next release.

My thought is that if we always have the latest, there won't be any dependency issues with patternfly. We can add examples and they will be immediately available for pf.org -- no more waiting until the next sprint to add patternfly examples to pf.org. 

Likewise, we should do this for angular-patternfly as well. I would love to have my about modal example (angular-patternfly implementation) available in pf.org for the end of this sprint, but I must wait for the following sprint to add it to pf.org.